pkg/server/apiextensions.go

```diff
@@ -67,16 +68,22 @@ func (c *inheritanceCRDLister) ListWithContext(ctx context.Context, selector lab
 	// Check for API inheritance
 	inheriting := false
 	inheritFrom := ""
-	// TODO(ncdc): for now, we are hard-coding "admin" as the logical cluster where all Workspace
-	// resources must live, at least for API inheritance to work.
-	const adminClusterName = "admin"
+	org := helper.OrganizationCluster
+	ws := cluster.Name
+
+	if cluster.Name != helper.OrganizationCluster {
```

Comment:
This logic seems like it should live in the helper - it's pretty domain specific (and something we do in the parse function as well)